### DISCUSSION TITLE
Uploaded files stay behind orphaned in minio bucket

### DISCUSSION THREAD

**Discussion by bjornclauw** [Budibase/budibase#5552] [Help] [answered]
Hi,

I upload files through REST api to a custom service.
For this I use the 'attachment' component.

>new form (type: CREATE // schema: CUSTOM)
>>new attachment

After getting my `Image.url` from my component I use a REST API query to send my image.
After the query is finished I clear my form.

Turns out the attachments stay behind in the minio database. Am i using this component in the wrong way?

**Comment by melohagan** [comment, marked as answer]
Hi @bjornclauw 

This seems to be a bug to me, which I've raised here: https://github.com/Budibase/budibase/issues/5564

**Reply by bjornclauw** [reply]
Thankx @melohagan

Solving this at the moment with a cron job that empties the folder 'attachments' in the bucket every sunday. (need to make sure you don't have other resources for your app stored in there)
